### PR TITLE
Remove Dotnet.exe from shared fx

### PR DESCRIPTION
**Pull request by ramarag**
fixes https://github.com/dotnet/core-setup/issues/2233

### CODE REVIEW

**Comment by ramarag** [comment]
@gkhanna79 @eerhardt @chcosta PTAL

File: src/sharedFramework/sharedFramework.proj [hunk 1]
```diff
@@ -106,6 +105,19 @@
...
+      <TrimPkgsFromDeps Include="runtime.$(PackageTargetRid).microsoft.netcore.dotnetapphost" />
+      <TrimPkgsFromDeps Include="runtime.$(PackageTargetRid).microsoft.netcore.dotnethostresolver" />
+      <TrimPkgsFromDeps Include="microsoft.netcore.dotnetapphost" />
+      <TrimPkgsFromDeps Include="microsoft.netcore.dotnethostresolver" />
+      <TrimPkgsWithCmdPrefix Include="@(TrimPkgsFromDeps ->'--remove %(Identity)')" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <TrimPkgsCmd>@(TrimPkgsWithCmdPrefix)</TrimPkgsCmd>
+      <TrimPkgsCmd>$(TrimPkgsCmd.Replace(';', ' '))</TrimPkgsCmd>
```
Comment (eerhardt) [inline src/sharedFramework/sharedFramework.proj]: You don't need to do it this way.  Instead you can do something like the following:

`<RestoreSourceArg>@(RestoreTestSource->'--source %(Identity)', ' ')</RestoreSourceArg>`

which puts the space as a separator from the start.

Comment (ramarag) [inline src/sharedFramework/sharedFramework.proj]: great tip, did not know that

**Comment by ramarag** [comment]
The signing is tracked by a separate issue https://github.com/dotnet/core-setup/issues/2231

**Pull request merged**